survey.html

```diff
@@ -31,31 +31,57 @@
             <main>
                 <h1>Survey</h1>
                 <form action="https://formspree.io/f/mnqopplo" method="POST">
-                    <label for="id">First Name:</label>
-                    <input type="text" id="fname" name="fname"><br>
-                    <label for="email">Enter your email:</label>
-                    <input type="email" id="email" name="email"><br>
-                    <label for="quantity">What is your favourite number?</label>
-                    <input type="number" id="quantity" name="quantity" min="1" max="999999999"><br>
-                    <label for="day">Which day is best?</label>
-                    <input type="date" id="day" name="day"><br>
-                    <label for="bear">Which bear is best?</label>
-                    <select name="bear" id="bear">
-                        <option value="Black">Black Bear</option>
-                        <option value="Grizzly">Grizzly Bear</option>
-                        <option value="Polar">Polar Bear</option>
-                        <option value="Sun">Sun Bear</option>
-                        <option value="Moon">Moon Bear</option>
-                    </select><br>
-                    <input type="radio" id="yes" name="opinion" value="yes">
-                    <label for="yes">Yes</label><br>
-                    <input type="radio" id="no" name="opinion" value="no">
-                    <label for="no">No</label><br>
-                    <input type="radio" id="?" name="opinion" value="?">
-                    <label for="?">None of your Business</label><br>
-                    <label for="life">What is the meaning of life?:</label>
-                    <textarea id="life" name="life" rows="5" cols="50"></textarea>
-                    <input type="submit" value="Submit">
```

Comment:
Lots of missing stuff here.  
Controls were not wrapped in `<p>` elements as per lab instructions.
Labels for required fields were missing the `<abbr>` markup for the * as per lecture
Missing `required` attributes